lib/services/plugins/cordova-project-plugins-service.ts

```diff
@@ -235,36 +188,43 @@ export class CordovaProjectPluginsService implements IPluginsService {
 
 	public removePlugin(pluginName: string): IFuture<void> {
 		return (() => {
-			if(!pluginName) {
+			if (!pluginName) {
 				this.$errors.fail("No plugin name specified.");
 			}
 
 			let installedPlugins = this.getInstalledPluginByName(pluginName);
 			let plugin = installedPlugins[0];
-			if(!plugin) {
-				this.$errors.fail("Could not find plugin with name %s.", pluginName);
+			if (!plugin) {
+				// Need to check the plugins directory because the plugin can be fetched, not added.
+				if (this.isPluginFetched(pluginName).wait()) {
+					this.$fs.deleteDirectory(path.join(this.$project.projectDir, this.getPluginsDirName(), pluginName)).wait();
```

Comment:
Sounds like a good idea - however I wouldn't insist on it, because after all the user has stated that he wants the plugin removed by running `$ appbuilder plugin remove`.
On the other hand asking before deletion is always a good idea